web/src/message_view.js

```diff
@@ -842,6 +839,7 @@ export function fast_track_current_msg_list_to_anchor(anchor) {
                 return msg_list_data.first().id;
             },
         });
+        message_lists.current.unfocus_navigation_bar();
```

Comment:
Hmm, why is this necessary? I'd think that changing the selected message would do this?

Reply:
Yeah, moved this logic to `message_selected.zulip` event handler in message_scroll instead.

```js
if (event.force_rerender && event.id === event.msg_list.last()?.id) {
    event.msg_list.focus_navigation_bar();
} else {
    event.msg_list.unfocus_navigation_bar();
}
```